pbxplore/analysis/visualization.py

```diff
@@ -162,7 +162,7 @@ def plot_map(fname, count_mat, idx_first_residue=1, residue_min=1, residue_max=N
     # add ticks and labels
     ax2.set_xticks(xticks - numpy.min(xticks))
     ax2.set_xticklabels(xticks)
-    ax2.set_yticks(range(len(yticks)))
+    ax2.set_yticks(list(range(len(yticks))))
```

Comment:
Does this need to be a list or is matplotlib happy with an iterator. I guess it was happy with the iterator otherwise it would not have worked on python 3 before.

Reply:
Indeed, 2to3 artifact, solved it